gradle/eclipse-project-layout.gradle

```diff
@@ -4,6 +4,7 @@
  */
 
 def isTestProject = name.endsWith('tests')
+def isPlugin = new File("$projectDir/META-INF/MANIFEST.MF").exists()
```

Comment:
am not sure if this is the common pattern or `project.file("META-INF/MANIFEST.MF")`

Reply:
I don't know either. Should I change this?

Reply:
i think we can keep it